package/harvester-os/files/etc/systemd/system/cos-setup-initramfs.service.d/override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/usr/sbin/migration.sh
```

Comment:
Prefix an `-` sign to ignore errors.
What's the consequence if the migration fails? I'd prefer still boot the system so users can fix it.

Reply:
Currently, we don't have a problem if the migration fails.
But if there is more script in the future.
We should consider dealing with it.

I will change it to ignore error in this stage